packages/lesswrong/lib/routes.ts

```diff
@@ -582,12 +582,14 @@ if (hasEventsSetting.get()) {
       name:'Localgroups.single',
       path: '/groups/:groupId',
       componentName: 'LocalGroupSingle',
+      titleComponentName: 'LocalgroupPageTitle',
```

Comment:
I'm not sure why we have separate components to handle page titles, but I figured I should follow the convention.